### PR TITLE
Building on linux

### PR DESCRIPTION
**Pull request by edisionnano**

### CODE REVIEW

**Comment by ameerj** [comment]
We can probably remove this header in main. Does building on linux complain about the libusb include in `input_common/gcadapter/gc_adapter.h`?

**Comment by edisionnano** [comment]
Yes it does you have to replace it there too

**Comment by ameerj** [comment]
@edisionnano can you please try locally merging #4280 and seeing if that fixes your issue?

File: src/input_common/main.cpp [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 #include <memory>
 #include <thread>
-#include <libusb.h>
+#include <libusb-1.0/libusb.h>
```
Comment (jbeich) [inline src/input_common/main.cpp:7]: Breaks build on DragonFly and FreeBSD:
```c++
src/input_common/main.cpp:7:10: fatal error: 'libusb-1.0/libusb.h' file not found
#include <libusb-1.0/libusb.h>
         ^~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

Comment (jbeich) [reply src/input_common/main.cpp:7]: Note, libusb is supported but it's different from [the reference implementation](https://github.com/libusb/libusb):
```
$ ls -1d /usr/include/*usb*
/usr/include/libusb.h
/usr/include/libusb20.h
/usr/include/libusb20_desc.h
/usr/include/usb.h
/usr/include/usbhid.h
```

**Comment by ameerj** [comment]
Please pull the latest master and see if that resolves your build errors

**Comment by ReinUsesLisp** [comment]
This seems to be fixed in latest master. Please comment here if it still happens. Closing.

**Pull request closed without merging**